lib/synapse-plugins/meson.build

```diff
@@ -0,0 +1,26 @@
+synapse_plugins_sources = [
+    'calculator-plugin.vala',
+    'command-plugin.vala',
+    'desktop-file-plugin.vala',
+    'switchboard-plugin.vala',
+    'system-managment.vala',
+    'link-plugin.vala',
+    'appcenter-plugin.vala'
+    ]
+
+synapse_plugins_dir = include_directories('.')
+
+synapse_plugins = static_library('synapse-plugins',
+                        synapse_plugins_sources,
+                        link_with : synapse_core,
```

Comment:
no need for link_with and include_directories of synapse_core, just add synapse_core_dep in the `dependencies` list